tmt/steps/execute/internal.py

```diff
@@ -117,7 +117,6 @@ def go(self):
         for guest in self.step.plan.provision.guests():
 
             # Push workdir to guest and execute tests
-            guest.push()
```

Comment:
I had to move push to guest to earlier stage, so the worktree is available for the prepare step.